checkstyle/suppressions.xml

```diff
@@ -340,6 +340,10 @@
     <suppress checks="CyclomaticComplexity"
               files="ShareCoordinatorShard.java"/>
 
+    <!-- share -->
+    <suppress checks="ClassDataAbstractionCoupling"
```

Comment:
I'd prefer to see this as an annotation in the class `@SuppressWarnings("ClassDataAbstractionCoupling")`.